kolibri/logger/csv.py

```diff
@@ -47,10 +47,10 @@ def get_content_title(self, obj):
             return ""
 
     def get_time_spent(self, obj):
-        return round(obj.time_spent, 3)
+        return "{:10.1f}".format(round(obj.time_spent, 1))
 
     def get_progress(self, obj):
-        return round(obj.progress, 1)
+        return "{:10.4f}".format(round(obj.progress, 4))
```

Comment:
Not much of an issue as it will just pad with leading `0`s, but as progress is only ever between `0` and `1`, and we only ever have 4 numbers after the decimal point, having at least 10 characters overall seems like overkill :)